api_specification/api_specification.json

```diff
@@ -0,0 +1,91 @@
+{
+  "swagger" : "2.0",
+  "info" : {
+    "description" : "The api starts with `/v1/<contract-category>/`\neach category will have it's interface.\n- The only required parameters for `DIGITS` are `shortcode` + `currency`.\n- We have explicitly decided not to support back pricing in this endpoint.\n- A future endpoint will have an extra date_pricing parameter, to support back pricing.\n",
+    "version" : "1.0.0",
+    "title" : "Pricing Endpoint"
+  },
+  "host" : "virtserver.swaggerhub.com",
+  "basePath" : "/weisheng-binary/test/1.0.0",
+  "tags" : [ {
+    "name" : "v1",
+    "description" : "Retrieval of bid and ask price"
+  } ],
+  "schemes" : [ "https", "http" ],
+  "paths" : {
+    "/v1/DIGITS/{shortcode}/{currency}" : {
+      "get" : {
+        "tags" : [ "v1" ],
+        "summary" : "Price the DIGITS contract",
+        "description" : "Computation of ask price and bid price of DIGITS contract",
+        "operationId" : "digits",
+        "produces" : [ "application/json", "application/xml" ],
+        "parameters" : [ {
+          "name" : "shortcode",
+          "in" : "path",
+          "description" : "Shortcode",
+          "required" : true,
+          "type" : "string"
+        }, {
+          "name" : "currency",
+          "in" : "path",
+          "description" : "Currency",
+          "required" : true,
+          "type" : "string"
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "successful operation",
+            "schema" : {
+              "type" : "array",
+              "items" : {
+                "$ref" : "#/definitions/v1"
```

Comment:
Good use of definition, however v1 isn't a good entity.
I'd say that a good example of a reference is the #/definitions/price which both ask_price and bid_price reference.

What's the array for here, btw?